styles/gauge.mapcss

```diff
@@ -0,0 +1,1134 @@
+/*
+OpenRailwayMap Style File for JOSM and KothicJS
+Gauge layer
+
+OpenRailwayMap Copyright (C) 2021
+This program comes with ABSOLUTELY NO WARRANTY.
+This is free software, and you are welcome to redistribute it under certain conditions.
+See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
+
+Format details:
+* https://josm.openstreetmap.de/wiki/Help/Styles/MapCSSImplementation
+* https://wiki.openstreetmap.org/wiki/MapCSS/0.2
+*/
+
+meta
+{
+	title: "OpenRailwayMap Gauge layer";
+	version: "0";
+	description: "Style for a gauge layer railroad network map.";
+	author: "TrainTraveler";
+	link: "http://wiki.openstreetmap.org/wiki/OpenRailwayMap";
+	watch-modified: true;
+}
+
+canvas
+{
+	default-points: true;
+	default-lines: true;
+}
+
+@import url("common.mapcss");
+
+/* extend common tracks with monorail and miniature */
+
+way|z10-[railway=monorail]
+{
+  set .mono_tracks_present;

```

Comment:
Please indent these lines with 2 tabs as the other styles and the lines above also use.

Please also make sure to use Unix line endings, i.e. LF only, as the file currently has CRLF line endings.

Reply:
done